src/dynamo.js

```diff
@@ -46,4 +48,28 @@ export class Dynamo {
     let params = { RequestItems: { OnlineUsers: userDeleteRequests } }
     await client.batchWriteItem(params).catch((e) => console.error('Error while deleting stale users', e))
   }
-}
\ No newline at end of file
+
+  async createGame(participants) {
+    const id = crypto.randomUUID()
+    const params = {
+      TableName: tableGames,
+      Item: marshall({
+        id: id,
+        participants: participants,
+        playerTurn: participants[Math.floor(Math.random())],
+        state: 'FINISHED'
```

Comment:
When working with predefined values like game state it makes sense to use enums. 
Pure enums in JS don't exist, so we can do something like:
```js
//constants.js
export const gameState = {
STARTED:'STARTED',
FINISHED:'FINISHED'
}
// dynamo.js
const {STARTED, FINISHED} = gameState
...
Item:{
//...
state: STARTED
}
```